src/lib/validations/Core_RunFlowBlockValidator.ts

```diff
@@ -0,0 +1,16 @@
+import {IBlock} from '@floip/flow-runner'
+import {ExtendedValidatorBase, ValidationResults} from './ExtendedValidatorBase'
+
+export default class Core_RunFlowBlockValidator extends ExtendedValidatorBase {
+  static runProgrammaticValidations(block: IBlock, options?: unknown): ValidationResults | null {
+    const {flow_id} = block.config
+
+    if (typeof flow_id !== 'string' || flow_id.length === 0) {
```

Comment:
as flow_id is a REQUIRED string https://github.com/FLOIP/flow-runner/blob/master/src/model/block/IRunFlowBlockConfig.ts#L23, why don't we provide `undefined` as value when it's empty instead of `empty string`. That way we don't need a programmatic validation.
Your thoughts?

Reply:
Maybe, yes, thanks Safydy!

Could you maybe help a bit?
If I'm doing this:
```
async createWith(...) {
    ...
    props.config = {
      flow_id: null,
    }
    ...
  }
```
I'm getting this:
<img width="956" alt="Screen Shot 2022-07-26 at 4 37 37 PM" src="https://user-images.githubusercontent.com/18130716/181035797-c5be73df-14f9-4206-9ebd-1d0119be35ff.png">

Reply:
If I'm doing this
```
props.config = {
      flow_id: undefined,
}
```
or just `props.config = {}`
then I'm getting this
<img width="813" alt="Screen Shot 2022-07-26 at 4 41 58 PM" src="https://user-images.githubusercontent.com/18130716/181036361-87419870-9026-404f-ba19-bbd34ad0dd08.png">

Reply:
The second one is worse, I think. Because its path is `/config` which is not too specific and it's not showing the red box.

And in the first case it's `/config/flow_id` which is specific and which shows the red box.

Reply:
I wonder how we can customise the validation message...

Reply:
Trying all this here:
https://github.com/FLOIP/flow-builder/pull/295/files